project/images/templates/images/form_source.html

```diff
@@ -77,6 +77,25 @@
 
         {% include "form_generic_one_field.html" with field=sourceForm.confidence_threshold dont_show_help_text="True" %}
     {% endif %}
+
+    {% if source_form_type == "new" or source.enable_robot_classifier %}
+        <hr class="light"/>
+
+        <legend class="smaller">Feature extractor</legend>
+
+        <span class="helptext_long">
+        {# TODO #}
+        (TODO: Explain what this is and how to choose it.)
```

Comment:
Suggested text:

We recommend EfficientNet extractor for all use-cases. It is faster and 2-3% more accurate on average. It is a more modern neural network architecture and trained on more data. The legacy VGG16 extractor is provided only for sources that want to retain their old classifiers, for example, if they are already deployed in a survey.

Reply:
Sounds good, added a commit for that.